### DISCUSSION TITLE
Can start Windows XP in newer UTM

### DISCUSSION THREAD

**Discussion by jesussmile** [utmapp/UTM#3426] [UTM Help (iOS)]
**Describe the issue**
I can run windows Xp fine in UTM 2.1.2 but in the latest version I get `press ESC in 5 seconds to skip startup.nsh or any other key to continue.
Shell>`

how can I hit the ESC, no keyboard supports it. Just cant get past this screen.
[text.txt](https://github.com/utmapp/UTM/files/7796747/text.txt)

**Configuration (required)**
* UTM Version: 2.4.1
* OS Version: 14.7.1
* Device Model: iPad Pro 12.9 first generation ML0G2ZA/A
* Is it jailbroken (name jailbreak used)? Yes, checkra1n
* How did you install UTM? GitHub Filza

**Crash log**
Nil

**Debug log**
Nil

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment]
Are you sure you didn’t create a new VM or changed the settings? Anyways you need to disable EFI in system->advanced

**Reply by jesussmile** [reply]
> Are you sure you didn’t create a new VM or changed the settings? Anyways you need to disable EFI in system->advanced

I used it as is,  It is on by default.

**Reply by osy** [reply]
Right it was newly added in v2.X. VMs created before then don't have it on by default. VMs created after you need to toggle it off.

**Reply by Sinemora10** [reply]
Not sure if it is helpful or even relevant, but it should be possible to remap the function of the caps lock key on the keyboard to ‘escape’ in the iPad settings, assuming you are using the apple magic keyboard. Otherwise, connecting a Bluetooth keyboard with the ESC key may do the trick?
